src/cloud-api-adaptor/docs/SecureComms.md

```diff
@@ -2,31 +2,86 @@
 
 Here, we describe how to set up the Secure Communications (`Secure Comms` for short) feature of the `cloud-api-adaptor` (`Adaptor`) and `agent-protocol-forwarder` (`Forwarder`).
 
-The `Secure Comms` feature establishes an SSH channel by the `Adaptor` as an SSH client and the `Forwarder` as an SSH server. All control communications then use this SSH channel between the `Adaptor` and `Forwarder`. Secure Comms use a two-phase approach:
+The `Secure Comms` feature establishes an SSH channel by the `Adaptor`, as an SSH client, and the `Forwarder`, as an SSH server. All control communications then use this SSH channel between the `Adaptor` and `Forwarder`. This feature is an alternative to using `TLS encryption` feature between the `Adaptor` and `Forwarder`. 
+
+Both `Secure Comms` and `TLS encryption` supports a method of delivering keys to encrypt the control-plane traffic between the `Adaptor` and the `Forwarder` via user-data during pod VM creation. Delivering the keys via the user-data may allow a malicious entity on the cloud provider side to gain access to the keys and modify any communication between the WNs and PPs. For this reason, `Secure Comms` also supports an alternate mode where the keys are delivered to the PP via `Trustee`. The keys are delivered to the PP after an attestation phase (See diagram below). Delivering the keys via `Trustee` protects the PP from a malicious entity on the cloud provider side. 
```

Comment:
Can we replace WN for worker node and PP for peer pods please to help with clarity?

Reply:
IMHO this would also improve the readability of the code too.